src/Microsoft.Identity.Web.OWIN/OwinTokenAcquisitionHost.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Net;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using System.Web;
+using Microsoft.Extensions.Options;
+using Microsoft.Identity.Client;
+using Microsoft.IdentityModel.JsonWebTokens;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Microsoft.Identity.Web.Hosts
+{
+    internal class OwinTokenAcquisitionHost : ITokenAcquisitionHost
+    {
+
+        IOptionsMonitor<MicrosoftIdentityOptions> _microsoftIdentityOptionsMonitor;
+        IOptionsMonitor<MergedOptions> _mergedOptionsMonitor;
+        IOptionsMonitor<ConfidentialClientApplicationOptions> _ccaOptionsMonitor;
+
+        public OwinTokenAcquisitionHost(
+            IOptionsMonitor<MicrosoftIdentityOptions> optionsMonitor, 
```

Comment:
>optionsMonitor [](http://example.com/codeflow?start=54&length=14)

better to call these MicrosoftIdentityOption?

Reply:
yes!

Reply:
updated.